js/main.js

```diff
@@ -0,0 +1,73 @@
+// var donate = document.querySelector('#donateBike');
+// donate.addEventListener('click', showTheAlert);
+
+// function showTheAlert() {
+//     alert('Donate button clicked');
+// }
+/*var change = document.querySelector('#changeButtons');
+change.addEventListener('click', changeAll);
+
+function appendParagraph() {
+    var paragraph = document.createElement('p');
+    paragraph.innerText = "How are you?";
+    var main = document.querySelector('#mainArticles');
+    main.appendChild(paragraph);
+}
+function changeAll() {
+    changeTheColor();
+    appendParagraph();
+    changeParagraphColor();    
+}
+
+function changeParagraphColor(){
+    var paragraphs = document.querySelectorAll('p');
+    for(var i=0; i<paragraphs.length; i++){
+        paragraphs[i].style.backgroundColor = 'blue';
+    }
+}
+changeParagraphColor();
+
+*/
+
+
+var donate = document.querySelector('#donateBike');
+donate.addEventListener('click', changeTheColor);
+
+function changeTheColor(event) {
+    event.preventDefault();
+    var myJumbo = document.querySelector('.jumbotron');
+    myJumbo.className += " highlites";
+}
+
+
+var addArticle = document.querySelector('#addArticleBtn');
```

Comment:
I assume all of this is an extra. I am not sure what is the full intention so will no code review further but do follow the same principles as before.